src/main/java/org/apache/sysds/runtime/controlprogram/federated/monitoring/models/NodeEntityModel.java

```diff
@@ -68,13 +83,43 @@ public void setStats(final List<BaseEntityModel> stats) {
 		_stats = stats;
 	}
 
+	public double getJitCompileTime() {
+		return _jitCompileTime;
+	}
+
+	public void setJitCompileTime(final double jitCompileTime) {
+		_jitCompileTime = jitCompileTime;
+	}
+
+	public String getRequestTypeCount() {
+		List<String> requestTypeCountStrArr = new ArrayList<>();
+
+		if (_requestTypeCount != null) {
+			for(var entry : _requestTypeCount) {
+				requestTypeCountStrArr.add(String.format("{" +
```

Comment:
this is not nice, make the formatting string outside the for loop (i do not trust the java compiler to fix this.)

Reply:
Done, moved logic into a separate file.